api-reference/media-api/list-and-search-files.md

```diff
@@ -265,7 +278,7 @@ Refer to the [supported parameters](list-and-search-files.md#supported-parameter
 
 | Field | Supported Operators | Examples|
 | ----------------------------------------------------------------------- | --------- | ---------------------- |
-|<p>  name                                   </p>| <ul><li>=</li><li>:</li><li>IN</li><li>NOT =</li><li>NOT IN</li></ul>                                                                                                                 | <p>Accepts a string value in quotes. For example:</p><p><code>name = "red-dress.jpg" </code>will return all<br>files &#x26; folders with the exact name <code>red-dress.jpg</code>.</p><p><code>name: "red-dress"</code> will return all files &#x26; folders</p><p>with a name starting with <code>red-dress</code>.</p><p><code>name IN ["red-dress.jpg", "red-dress.png"]</code> will return</p><p>all files &#x26; folders with the name either <code>red-dress.jpg</code> or</p><p><code>red-dress.png</code>.</p><p><code>name NOT = "red-dress.jpg"</code> will return all files and folders</p><p>with a name other than <code>red-dress.jpg</code>.</p>                                                                                                                                       |
+|<p>  name                                   </p>| <ul><li>=</li><li>:</li><li>IN</li><li>NOT =</li><li>NOT IN</li><li>HAS</li></ul>                                                                                                                 | <p>Accepts a string value in quotes. For example:</p><p><code>name = "red-dress.jpg" </code>will return all<br>files &#x26; folders with the exact name <code>red-dress.jpg</code>.</p><p><code>name: "red-dress"</code> will return all files &#x26; folders</p><p>with a name starting with <code>red-dress</code>.</p><p><code>name IN ["red-dress.jpg", "red-dress.png"]</code> will return</p><p>all files &#x26; folders with the name either <code>red-dress.jpg</code> or</p><p><code>red-dress.png</code>.</p><p><code>name NOT = "red-dress.jpg"</code> will return all files and folders</p><p>with a name other than <code>red-dress.jpg</code>.</p><p><code>name HAS "red"</code> will return all files and folders starting with the word <code>red</code> or containing the word <code>red</code>.</p>                                                                                                                                      |
```

Comment:
Be more clear about `containing` here. We should convey about the delimiter based behaviour.

Also update details in [this page](https://docs.imagekit.io/media-library/overview/search-update-and-delete).